src/background/modules/AutocorrectHandler.js

```diff
@@ -43,8 +46,9 @@ function createRegEx(tree) {
         if (char) {
             const escaptedChar = char.replace(regExSpecialChars, "\\$&");
 
-            if (!("" in tree[char] && Object.keys(tree[char]).length === 1)) {
-                const recurse = createRegEx(tree[char]);
+            const atree = tree[char];
+            if (!(LEAF in atree && Object.keys(atree).length === 0)) {
```

Comment:
IMHO this looks really more readable now, does not it? Thanks a lot.

Reply:
Yes, I agree it is a marginal improvement...